src/pages/public_registration/PublicProjectsContent.tsx

```diff
@@ -20,25 +32,50 @@ const PublicProjectsContent = ({ registration }: PublicRegistrationContentProps)
       </Typography>
 
       <MuiThemeProvider theme={lightTheme}>
-        {registration.projects.map((project) => (
-          <StyledProjectChip
-            key={project.id}
-            label={
-              <Typography>
-                {project.name}
-                {project.grants && project.grants.length > 0 && (
-                  <span>
-                    {' '}
-                    ({t('description.financed_by')}: {project.grants.map((grant) => grant.source).join(', ')})
-                  </span>
-                )}
-              </Typography>
-            }
-          />
+        {projects.map((project) => (
+          <ProjectRow key={project.id} project={project} />
         ))}
       </MuiThemeProvider>
     </>
   );
 };
 
-export default PublicProjectsContent;
+interface ProjectRowProps {
+  project: ResearchProject;
+}
+
+const ProjectRow = ({ project }: ProjectRowProps) => {
+  const { t } = useTranslation('registration');
+  const [fetchedProject, isLoadingProject] = useFetch<CristinProject>(project.id, true); // TODO: remove auth
```

Comment:
Prosjekt api krever autentisering atm. når det er fikset:
```suggestion
  const [fetchedProject, isLoadingProject] = useFetch<CristinProject>(project.id);
```